### PR TITLE
Adjusted Header display renderer.

### PR DESCRIPTION
**Pull request by Robert-Lark**
# Description
This PR takes care of the code needed to ensure that the child header is rendered only on the child pages and the parent header is rendered only on the parent pages. 

Fixes #

- What work was done?
Parent Header is in its own file "ParentNavTop" so to follow this convention I separated the Child Header into its own common component also. These were then brought into a single Header common component and are rendered based off the URL pathname. If the pathname includes Parent it will render the parent header. If not it will render the child header. 
- Why was this work done?
Following the figma design
- What feature / user story is it for?
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/common/Childnav.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  const showNav = () => {
+    switch (history.location.pathname) {
+      case '':
+      case '/login':
+        return false;
+      default:
+        return true;
+    }
+  };
+
```
Comment (Robert-Lark) [inline src/components/common/Childnav.js:68]: this function was left alone as it only relates to the hamburger menu on child pages and works well eliminating the hamburger on login pages.

File: src/components/common/Childnav.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```
Comment (Robert-Lark) [inline src/components/common/Childnav.js:1]: this is a new component, housing all the code from the Header.js file. Header.js now just holds a function that renders headers based off the current URL pathname.

File: src/components/pages/AddChild/RenderAddChild.js [hunk 3]
```diff
@@ -12,7 +9,6 @@ const { Title } = Typography;
 const RenderAddChild = props => {
   return (
     <Layout className="add-child">
```
Comment (Robert-Lark) [inline src/components/pages/AddChild/RenderAddChild.js:11]: All of the following changes are removing the parentNavTopBar as it no longer needs to be called in the individual components.

File: src/components/common/Header.js [hunk 4]
```diff
@@ -1,86 +1,29 @@
...
-      case '':
-      case '/login':
-        return false;
-      default:
-        return true;
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import ChildHeader from './ChildNav';
+import ParentNavTop from './ParentNavTop';
+
```
Comment (Robert-Lark) [inline src/components/common/Header.js:5]: Header Renderer below takes care of the Headers on every page - it looks at the URL pathname and renders a header (stored in state) based off the pathname.

**Review by depadiernos — DISMISSED** [review]
Lgtm

**Review by Koyamot — APPROVED** [review]
Looks Great! Good job!!!

**Pull request merged**